src/main/java/com/cloudblue/connect/api/models/enums/CBCCaseState.java

```diff
@@ -0,0 +1,5 @@
+package com.cloudblue.connect.api.models.enums;
+
+public enum CBCCaseState {
+    pending, inquiring, resolved, closed
```

Comment:
This will cause code smell on sonar report. Fix as we have done for other enums.

Reply:
Done